package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Think maybe we should maybe just remove anything that's not used from here: `directories`, `devDependencies`, `scripts`, `keywords`, `author`.